Documentation/CMake.md

```diff
@@ -59,21 +59,38 @@ endif()
 ## Add an entry point
 
 You must include a source file in your test executable target with a
-`@main` entry point. The following example uses the SwiftPM entry point:
+`@main` entry point. The example main below requires the experimental
+`Extern` feature. The function `swt_abiv0_getEntryPoint` is exported
+from the swift-testing dylib. As such, its declaration could instead
+be written in a C header file with its own `module.modulemap`, or
+the runtime address could be obtained via
+[`dlsym()`](https://pubs.opengroup.org/onlinepubs/9799919799/functions/dlsym.html) or
```

Comment:
You already changed the URL, but for future reference, official Linux documentation is on kernel.org, and man7.org is technically one (important-to-Linux) engineer's personal website. (kernel.org just links out to man7.org, but still...) 😄